bin/bootstrax

```diff
@@ -660,7 +660,7 @@ def set_state(state, update_fields=None):
     )
     if update_fields:
         bootstrax_state.update(update_fields)
-    if (not previous_entry or
+    if (previous_entry is None or
             (now() - previous_entry['_id'].generation_time).seconds > timeouts[
```

Comment:
`previous_entry` should contain a `time` field, we can maybe improve the readability if we use it
```suggestion
            (now() - previous_entry['time'].replace(tzinfo=pytz.utc).seconds > timeouts[
```

Reply:
Thanks for the suggestion!